packages/manager/apps/pci-savings-plan/src/components/CreatePlanForm/CreatePlanForm.tsx

```diff
@@ -104,6 +104,7 @@ export type CreatePlanFormProps = {
     size: number;
   }) => void;
   isDiscoveryProject: boolean;
+  isPendingCreatePlan: boolean;
```

Comment:
I would have rather call it `isCreatePlanPending` because it sounds better to me, but it's up to you 😌